x-pack/plugins/cloud_security_posture/server/routes/compliance_dashboard/get_clusters.ts

```diff
@@ -14,7 +14,7 @@ import type {
 import { Cluster } from '../../../common/types';
 import { getResourceTypeFromAggs, resourceTypeAggQuery } from './get_resources_types';
 import type { ResourceTypeQueryResult } from './get_resources_types';
-import { CSP_KUBEBEAT_INDEX_PATTERN } from '../../../common/constants';
+import { LATEST_FINDINGS_INDEX_PATTERN } from '../../../common/constants';
```

Comment:
fyi: currently `LATEST_FINDINGS_INDEX_PATTERN` is referring only to a single index, and is not really a pattern. we would fix that in the future